android/app/src/main/java/poke/rogue/helper/presentation/dex/detail/PokemonDetailActivity.kt

```diff
@@ -16,28 +18,29 @@ import poke.rogue.helper.presentation.ability.AbilityActivity
 import poke.rogue.helper.presentation.base.toolbar.ToolbarActivity
 import poke.rogue.helper.presentation.battle.BattleActivity
 import poke.rogue.helper.presentation.biome.detail.BiomeDetailActivity
-import poke.rogue.helper.presentation.dex.PokemonTypesAdapter
 import poke.rogue.helper.presentation.home.HomeActivity
-import poke.rogue.helper.presentation.type.view.TypeChip
 import poke.rogue.helper.presentation.util.context.stringArrayOf
 import poke.rogue.helper.presentation.util.context.stringOf
 import poke.rogue.helper.presentation.util.context.toast
 import poke.rogue.helper.presentation.util.repeatOnStarted
 import poke.rogue.helper.presentation.util.view.dp
 import poke.rogue.helper.presentation.util.view.loadImageWithProgress
+import poke.rogue.helper.ui.component.PokeChip
+import poke.rogue.helper.ui.layout.PaddingValues
 
 class PokemonDetailActivity :
     ToolbarActivity<ActivityPokemonDetailBinding>(R.layout.activity_pokemon_detail) {
     private val viewModel by viewModel<PokemonDetailViewModel>()
-    override val toolbar: Toolbar
-        get() = binding.toolbarPokemonDetail
-
-    private lateinit var pokemonTypesAdapter: PokemonTypesAdapter
+    override val toolbar: Toolbar?
+        get() = null
 
     private lateinit var pokemonDetailPagerAdapter: PokemonDetailPagerAdapter
 
     private var isExpanded = false
 
+    private var fullNameChipSpecs: List<PokeChip.Spec> = emptyList()
+    private var iconOnlyChipSpecs: List<PokeChip.Spec> = emptyList()
```

Comment:
```suggestion
private var pokemonTypes: List<TypeUiModel> = emptyList()
```

포켓몬 타입을 멤버 변수로 두는건 어떠세요??